features/appmgt/org.wso2.carbon.appmgt.gateway.feature/src/main/resources/repository/deployment/server/synapse-configs/default/sequences/main.xml

```diff
@@ -0,0 +1,104 @@
+  ~  software distributed under the License is distributed on an
+  ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  KIND, either express or implied.  See the License for the
+  ~  specific language governing permissions and limitations
+  ~  under the License.
+  -->
+
+<!-- Default main sequence shipped with the WSO2 ESB -->
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="main">
+    <description>The default main sequence for API manager - Returns 404 Not Found</description>
+    <in>
+        <log level="custom">
+            <property name="STATUS" value="Message dispatched to the main sequence. Invalid URL."/>
+            <property name="RESOURCE" expression="fn:concat('/', $axis2:REST_URL_POSTFIX)"/>
+        </log>
+	<!--Removing Build sequence as it is of no use when used with PT Transport-->
+        <!--sequence key="_build_"/-->
+        <filter source="$ctx:TRANSPORT_DENIED" regex="true">
+            <payloadFactory>
+                <format>
+                    <am:fault xmlns:am="http://wso2.org/apimanager">
+                        <am:code>403</am:code>
+                        <am:type>Status report</am:type>
+                        <am:message>Forbidden</am:message>
+                        <am:description>Unsupported Transport [ $2 ]. The requested resource (/$1) is not available.</am:description>
+                    </am:fault>
+                </format>
+                <args>
+                    <arg expression="$axis2:REST_URL_POSTFIX"/>
+                    <arg expression="$ctx:IN_TRANSPORT"/>
+                </args>
+            </payloadFactory>
+            <property name="HTTP_SC" value="403" scope="axis2"/>
+            <property name="RESPONSE" value="true"/>
+            <header name="To" action="remove"/>
+            <property name="NO_ENTITY_BODY" scope="axis2" action="remove"/>
+            <property name="ContentType" scope="axis2" action="remove"/>
+            <property name="Authorization" scope="transport" action="remove"/>
+            <property name="Host" scope="transport" action="remove"/>
+            <property name="Accept" scope="transport" action="remove"/>
+            <send/>
+        </filter>
+        <filter source="get-property('MESSAGE_FORMAT')" regex="soap1[1-2]">
+            <then>
+                <property name="SOAP_FAULT_CODE" value="Server"/>
+                <makefault>
+                    <code expression="$ctx:SOAP_FAULT_CODE"/>
+                    <reason value="The requested endpoint is not available."/>
+                </makefault>
+            </then>
+            <else>
+                <class name="org.wso2.carbon.appmgt.custom.mediators.redirection.RefererRewriteMediator"/>
+                <header name="Location" expression="fn:concat($trp:referer,get-property('REST_FULL_REQUEST_PATH'))" scope="transport"/>
+                <property name="HTTP_SC" value="303" scope="axis2"/>                
```

Comment:
In here we don't need to consider HTTP method. It can be GET,PUT,POST or either DELETE. So shouldn't it be 303?